tests/test-addons.py

```diff
@@ -0,0 +1,53 @@
+from validators import validate_dns, validate_dashboard, validate_storage
+from utils import microk8s_enable, wait_for_pod_state, microk8s_disable
+from subprocess import Popen, PIPE, STDOUT
+
+class TestAddons(object):
+
+    def test_dns(self):
+        """
+        Sets up DNS addon and validates it works.
+
+        """
+        print("Enabling DNS")
+        microk8s_enable("dns")
```

Comment:
Is there a way to not disrupt the cluster after running tests? If I have a microk8s with dns and dashboard enabled and then run tests I will have both off at the end.

```
original_dns = microk8s_is_enabled("dns")
microk8s_enable("dns")
...
if original_dns == false:
  microk8s_disable("dns")
```

Reply:
I think this should be addressed when we offer the microk8s.status https://github.com/juju-solutions/microk8s/issues/37 . These tests are to be run in a "controlled environment". I would rather know if dns did not come up instead of skipping the test.